### PR TITLE
fix: domain issue

### PR DESCRIPTION
**Pull request by alcipir**

### CODE REVIEW

**Comment by alcipir** [comment]
New version of this PR deployed on: https://fix-domain-issue.paybutton.io

**Comment by alcipir** [comment]
New version of this PR deployed on: https://fix-domain-issue.paybutton.io

**Comment by alcipir** [comment]
New version of this PR deployed on: https://fix-domain-issue.paybutton.io

**Comment by alcipir** [comment]
New version of this PR deployed on: https://fix-domain-issue.paybutton.io

**Comment by alcipir** [comment]
New version of this PR deployed on: https://fix-domain-issue.paybutton.io

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "custom-server-typescript",
   "version": "1.0.0",
   "scripts": {
-    "dev": "next dev",
+    "dev": " BRANCH=teste next dev",
```
Comment (Klakurka) [inline package.json:5]: 'teste' or 'test'?

Comment (alcipir) [reply package.json:5]: oops, this was just for testing the environment variables, forgot about it.

**Pull request merged**